tests/testthat/test-read_aihw_mental_health.R

```diff
@@ -0,0 +1,67 @@
+# testing read_aihw_mental_health function (contains four functions)
+
+test_that("Read of Australian Institute of Health and Welfare - Mental Health Data works", {
+
+  care_episodes <- read_aihw_care_episodes()
+
+  expect_s3_class(care_episodes, 'data.frame')
+  expect_equal(class(care_episodes$value), 'numeric')
+  expect_equal(class(care_episodes$observation_date), 'character')       # probably should be 'Date'?
+
+  expect_equal(colnames(care_episodes), c("sa3_code",
+                                          "observation_date",
+                                          "value"))
+
+  emergency_presentations <- read_aihw_emergency_presentations()
```

Comment:
there are essentially 4 sets of tests here, it is better to split them into their own `test_that` functions. This gives better granularity for when a test fails. Usually, you only test one function (or one part of a function) within each `test_that`

Reply:
Great, thanks for the feedback. Noted.